README.md

```diff
@@ -414,15 +414,46 @@ $ clarinet integrate
 
 Make sure that you have a working installation of Docker running locally.
 
-### Deploy contracts to Testnet
+### Interacting with contracts deployed on Mainnet
 
-You can use Clarinet to deploy your contracts to the public Testnet environment for testing and
-evaluation on a blockchain. Use the following command:
+Composition and interactions between protocols and contracts are one of the key innovations in blockchains. Clarinet was designed to handle this sort of interactions.
+
+Before refering to contracts deployed on Mainnet, they should be explicitily be listed as a `requirement` in the manifest `Clarinet.toml`:
+
+```toml
+[project]
+name = "my-project"
+requirements = [
+  "SP2KAF9RF86PVX3NEE27DFV1CQX0T4WGR41X3S45C.bitcoin-whales.payout"
+]
+
+```
+
+From there, clarinet will be able to reconciliate the `contract-call` present in your local contracts, download and cache a copy of the required contracts, and use them during the execution of your tests suites, and all the different features available in `clarinet`.
```

Comment:
I'm not sure what you mean by "clarinet will be able to reconciliate the `contract-call` present in your local contracts" here. Can you word that differently?

Reply:
Thanks, any take on https://github.com/hirosystems/clarinet/pull/333/commits/c03549d1fa3804faa9fbd19f01e0933d0051ae0f?